app/src/main/java/io/github/droidkaigi/confsched2018/presentation/MainActivity.kt

```diff
@@ -65,7 +65,11 @@ class MainActivity : BaseActivity(), HasSupportFragmentInjector {
             true
         })
         if (savedInstanceState == null) {
-            binding.bottomNavigation.selectedItemId = R.id.navigation_sessions
+            if (intent.getStringExtra("shortcut") != null) {
```

Comment:
Can you use `when`. like this? Because we can add it ⌚️ 

```
when(intent.getStringExtra("shortcut")) {
  getString(R.string.bottom_navigation_favorite) -> binding.bottomNavigation.selectedItemId = R.id.navigation_favorite_sessions 
  else -> binding.bottomNavigation.selectedItemId = R.id.navigation_sessions
```

Reply:
Yes, please wait.

Reply:
I used `when` follow your comment.
But don't use string resource, because can't use string resource for shortcuts.xml extra tag.
It's OK?

Reply:
Thanks ! OK 🆒